jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -730,9 +731,31 @@ static Instant getCreationTime(String configuredCreationTime, ProjectProperties
           return Instant.now();
 
         default:
-          return DateTimeFormatter.ISO_DATE_TIME.parse(configuredCreationTime, Instant::from);
+          DateTimeFormatter formatter =
+              new DateTimeFormatterBuilder()
+                  .parseCaseInsensitive()
+                  .append(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
+                  // iso 8601 strict
```

Comment:
https://github.com/frohoff/jdk8u-dev-jdk/blob/master/src/share/classes/java/time/format/DateTimeFormatter.java#L992

Reply:
Aha. Then can't we start with `ISO_DATE_TIME`? If I'm not mistaken, we only need to add `.appendOffset("+HHMM", "+0000")` for the case without a colon. So, can't we do something like this?

```java
new DateTimeFormatterBuilder()
                .append(ISO_DATE_TIME)
                .optionalStart()
                .appendOffset("+HHMM", "+0000")
                .optionalStart()
                .toFormatter(...)
```

If that doesn't work, at least, I think we can base on `.appendOffsetId()` and add only `.appendOffset("+HHMM", "+0000")`?

Reply:
Yeah I think you're right,  I think we want to use `+HHmm` which covers `+09`, and `+0900`